dashboard/app/helpers/levels_helper.rb

```diff
@@ -102,11 +102,9 @@ def select_and_track_autoplay_video
     autoplay_video.summarize unless params[:noautoplay]
   end
 
-  def select_and_remember_callouts(always_show = false)
-    # Filter if already seen (unless always_show)
-    callouts_to_show = @level.available_callouts(@script_level).
-      reject { |c| !always_show && client_state.callout_seen?(c.localization_key) }.
```

Comment:
I think we need to do a migration here-- otherwise a user will see callouts that are recorded in the session state but not in local storage yet.

(Alternatively, we might declare "callout bankruptcy" and say that users will see the callouts possibly one more time, but we would want to run that idea past Alice.  I think it's not the best experience.)